### PR TITLE
RAM_oM: Deprecating UniformLoadSet and ContourLoadSet

### PR DESCRIPTION
**Pull request by JosefTaylor**
### NOTE: Depends on 

[BHoM PR 1229](https://github.com/BHoM/BHoM/pull/1229)
   
### Issues addressed by this PR

Closes [BHoM 1224](https://github.com/BHoM/BHoM/issues/1224)

### Test files

https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/RAM_Toolkit/183%20Promoting%20UniformLoadSet%20and%20ContourLoadSet?csf=1&web=1&e=TakdUr

### CODE REVIEW

**Comment by JosefTaylor** [comment]
@BHoMBot check versioning

**Comment by JosefTaylor** [comment]
@BHoMBot check versioning

**Comment by JosefTaylor** [comment]
@BHoMBot check versioning

File: RAM_Engine/Versioning42.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (FraserGreenroyd) [inline RAM_Engine/Versioning42.json:1]: This file needs to be named `Versioning_42.json` - note the `_` before the numbers is missing, this is preventing this file from being picked up by Verioning_Toolkit

**Review by FraserGreenroyd — DISMISSED** [review]

**Comment by JosefTaylor** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check versioning

**Comment by JosefTaylor** [comment]
@BHoMBot check compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check code-compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check core

**Comment by JosefTaylor** [comment]
@BHoMBot check compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check core

**Comment by JosefTaylor** [comment]
@BHoMBot check versioning

**Comment by JosefTaylor** [comment]
@BHoMBot check documentation-compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check core

**Comment by JosefTaylor** [comment]
@BHoMBot check compliance

**Comment by JosefTaylor** [comment]
@BHoMBot check installer

**Comment by maryannewachter** [comment]
@JosefTaylor Versioning does not appear to have worked for the contour load set (though it did for the uniform load set). I'm getting the generic "This is a magic component" message.

image

**Review by maryannewachter — APPROVED** [review]
Minus the versioning bug on contourLoadSet (now Issue #184 ), the following checks have been confirmed:

- contourLoadSets can be pushed into and pulled from RAM
- uniformLoadSets can be pushed into and pulled from RAM
- old uniformLoadSet components automagically convert to the new uniformLoadSet (no longer RAM specific).

**Pull request merged**